lib/api/apiUtils/object/storeObject.js

```diff
@@ -26,7 +26,9 @@ function checkHashMatchMD5(stream, hashedStream, dataRetrievalInfo, log, cb) {
             contentMD5,
         });
         log.trace('contentMD5 does not match, deleting data');
```

Comment:
Wait, sorry, just for clarification - you think there should be only one log `log.error`? Or a `log.error` and `log.trace`?